doc/mgr/telemetry.rst

```diff
@@ -55,9 +55,11 @@ the per-channel setting has no effect.)
     - troubleshoot issues with latency, throttling, memory management, etc.
     - monitor cluster performance by daemon
 
-The reported data does *not* contain any sensitive data. This means that the
-reported data does not include pool names, object names, object contents,
-hostnames, or device serial numbers.
+The reported data does *not* contain any sensitive data[Does the reported data
```

Comment:
@yaarith , Can you give us an answer to this question?

Reply:
The reported data does not include IP addresses.